src/decorators/__tests__/__snapshots__/union.decorator.test.ts.snap

```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`decorators Union generates a oneOf definition 1`] = `
+Object {
+  "components": Object {
+    "schemas": Object {
+      "Foo": Object {
+        "properties": Object {
+          "bar": Object {
+            "oneOf": Array [
```

Comment:
Here's proof that we can generate a `oneOf` definition, assuming we want to continue down this path.